benchmarks/stb_stbi_read_fuzzer/build.sh

```diff
@@ -15,13 +15,6 @@
 #
 ################################################################################
 
-# remove this line when this change is upstreamed
-sed '2d' $SRC/stb/tests/stb_png_read_fuzzer.cpp > $SRC/stb/tests/stbi_read_fuzzer.c
```

Comment:
I think you need to keep this line.

Reply:
Yep, I am thinking the creator meant when the change is upstreamed in stb's code or something